Source/WebCore/dom/Element.h

```diff
@@ -604,6 +604,8 @@ class Element : public ContainerNode {
     virtual bool isSliderThumbElement() const { return false; }
     virtual bool isHTMLTablePartElement() const { return false; }
 
+    virtual bool isDevolvableWidget() const { return false; }
```

Comment:
virtual functions might be slow, worth A/B testing perf benchmarks (especially Speedometer)